### PR TITLE
persons link by resource

### PR DESCRIPTION
**Pull request by rerowep**
* Persons are now linked in the document by references to there resources
(IDREF, GND, ...)

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- run setup and verify everythink works as before

## How to test?

- https://tree.taiga.io/project/rero21-reroils/us/1241
- https://tree.taiga.io/project/rero21-reroils/task/1465

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json [hunk 1]
```diff
@@ -349,7 +349,7 @@
               "$ref": {
                 "title": "MEF person reference",
                 "type": "string",
-                "pattern": "^https://mef.rero.ch/api/mef/.*?$",
+                "pattern": "^https://mef.rero.ch/api/.*/.*?$",
```
Comment (iGormilhit) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:352]: Is it possible to have a more specific rule here, something like `gnd or idref or rero or bnd`?

Comment (rerowep) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:352]: Actualy you can have mef, viaf, bnf, gnd, idref, rero

Comment (rerowep) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json:352]: Schould I replace `.*` with `bnd\|gnd\|idref\|rero`?

File: rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py [hunk 2]
```diff
@@ -36,28 +36,45 @@ def get_person_link(bibid, id, key, value):
     """Get MEF person link."""
     # https://mef.test.rero.ch/api/mef/?q=rero.rero_pid:A012327677
     prod_host = 'mef.rero.ch'
-    test_host = 'mef.test.rero.ch'
-    mef_url = 'https://{host}/api/mef/'.format(host=test_host)
-    if os.environ.get('RERO_ILS_MEF_URL'):
-        mef_url = os.environ.get('RERO_ILS_MEF_URL')
+    test_host = os.environ.get('RERO_ILS_MEF_HOST', 'mef.test.rero.ch')
```
Comment (jma) [inline rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py:39]: mef.test or mef.rero.ch?

Comment (rerowep) [reply rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py:39]: Yes

**Pull request merged**